client/src/main/java/org/apache/rocketmq/client/trace/AsyncTraceDispatcher.java

```diff
@@ -71,7 +71,7 @@
     private volatile ThreadLocalIndex sendWhichQueue = new ThreadLocalIndex();
     private String dispatcherId = UUID.randomUUID().toString();
     private String traceTopicName;
-    private AtomicBoolean isStarted = new AtomicBoolean(false);
```

Comment:
Here,you can look at another pr before,the pr's link is:https://github.com/apache/rocketmq/pull/694.If you make the isStarted variable to non-static,it will result in some issues in sending trace msgs to different clusters.